Library/Formula/pgloader.rb

```diff
@@ -0,0 +1,313 @@
+    url "http://beta.quicklisp.org/archive/postmodern/2014-11-06/postmodern-20141106-git.tgz"
+    sha1 "ae4596a9407308bc844b074032a374cdb56d0192"
+  end
+
+  resource "puri" do
+    url "http://beta.quicklisp.org/archive/puri/2010-10-06/puri-20101006-git.tgz"
+    sha1 "ed72ad901cd9c2ecb171692a87dc4c06956f0004"
+  end
+
+  resource "py-configparser" do
+    url "http://beta.quicklisp.org/archive/py-configparser/2013-10-03/py-configparser-20131003-svn.tgz"
+    sha1 "510ec60b1b60945e925fd2a821824e81db31a2e1"
+  end
+
+  resource "qmynd" do
+    url "http://beta.quicklisp.org/archive/qmynd/2015-03-02/qmynd-20150302-git.tgz"
+    sha1 "20b9e4dd0345fa55e4c6eb166e103e3a746e74b6"
+  end
+
+  resource "salza2" do
+    url "http://beta.quicklisp.org/archive/salza2/2013-07-20/salza2-2.0.9.tgz"
+    sha1 "b00c72e61092e69232c2aa46b2559e093b0a8f77"
+  end
+
+  resource "split-sequence" do
+    url "http://beta.quicklisp.org/archive/split-sequence/2012-07-03/split-sequence-1.1.tgz"
+    sha1 "ed93b5fb609b9a04e9d1315b8ade25b5007d712e"
+  end
+
+  resource "trivial-backtrace" do
+    url "http://beta.quicklisp.org/archive/trivial-backtrace/2012-09-09/trivial-backtrace-20120909-git.tgz"
+    sha1 "ec0fb62c1ad44eba64db7a55b98fa81186e33e39"
+  end
+
+  resource "trivial-features" do
+    url "http://beta.quicklisp.org/archive/trivial-features/2015-01-13/trivial-features-20150113-git.tgz"
+    sha1 "1f887b67b31db22594e05b713f531afe7fd2e377"
+  end
+
+  resource "trivial-garbage" do
+    url "http://beta.quicklisp.org/archive/trivial-garbage/2015-01-13/trivial-garbage-20150113-git.tgz"
+    sha1 "ee35bca2a1668cf33bbeeeb229d5f33a0562d705"
+  end
+
+  resource "trivial-gray-streams" do
+    url "http://beta.quicklisp.org/archive/trivial-gray-streams/2014-08-26/trivial-gray-streams-20140826-git.tgz"
+    sha1 "1b5fa153940feacda3b6bd9f78104e310b4a4124"
+  end
+
+  resource "trivial-utf-8" do
+    url "http://beta.quicklisp.org/archive/trivial-utf-8/2011-10-01/trivial-utf-8-20111001-darcs.tgz"
+    sha1 "c875192cc54c6101c57e6593723a84fe0498e6d9"
+  end
+
+  resource "uiop" do
+    url "http://beta.quicklisp.org/archive/uiop/2014-11-06/uiop-3.1.4.tgz"
+    sha1 "db611c8e74aece4716e8eb05ea58f4378cbe8de0"
+  end
+
+  resource "usocket" do
+    url "http://beta.quicklisp.org/archive/usocket/2013-07-20/usocket-0.6.1.tgz"
+    sha1 "7c93c389c95db51209364365bfaec53fbcec2153"
+  end
+
+  resource "uuid" do
+    url "http://beta.quicklisp.org/archive/uuid/2013-08-13/uuid-20130813-git.tgz"
+    sha1 "8d38cf0f436246791c243de48c6382a641450278"
+  end
+
+  def install
+    resources.each do |resource|
+      resource.stage buildpath/"lib"/resource.name
+    end
+
+    ENV["CL_SOURCE_REGISTRY"] = "#{buildpath}/lib//"
+    ENV["ASDF_OUTPUT_TRANSLATIONS"] = "/:/"
+    system "make", "pgloader-standalone", "BUILDAPP=buildapp"
+
+    bin.install "build/bin/pgloader"
+    man1.install "pgloader.1"
+  end
+
+  test do
+    system "pgloader --version | grep 'pgloader version'"
```

Comment:
Can this test be modified to do something more substantial than e.g. `--version` or `--help`? See `cmake.rb` for an example of an application formula with a good test and `tinyxml2.rb` for an example of a library formula with a good test. Thanks!